DuckDuckGo/Application/AppDelegate.swift

```diff
@@ -742,6 +744,13 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
         NotificationCenter.default.addObserver(self, selector: #selector(dataImportCompleteNotification(_:)), name: .dataImportComplete, object: nil)
     }
 
+    private func subscribeToInternalUserChanges() {
+        UserDefaults.appConfiguration.isInternalUser = internalUserDecider.isInternalUser
+
+        isInternalUserSharingCancellable = internalUserDecider.isInternalUserPublisher
+            .assign(to: \.isInternalUser, onWeaklyHeld: UserDefaults.appConfiguration)
```

Comment:
We're re-publishing the internal user state to the app configuration user defaults, so that agents can see it.  It's just a bool and setting it manually to true would only affect the agents.

This was discussed & approved here: https://app.asana.com/0/481882893211075/1208617860225205/f